MARIADB_MASTER_PORT is failing to configure custom master port.

Hi, when trying to use the MARIADB_MASTER_PORT option on slave servers, the default 3306 is always used when the container is running. In compose the configuration option is set like this:

`- MARIADB_MASTER_PORT=3309`
    
But in the logs the slave is still connecting to master on port 3306:

`2016-08-01  0:19:57 140347877411584 [ERROR] Slave I/O: error connecting to master 'user1_repl@10.0.0.2:3306' - retry-time: 60  retries: 86400  message: Access denied for user 'user1_repl'@'10.0.0.3' (using password: YES), Internal MariaDB error code: 1045`

--- a/Dockerfile
+++ b/Dockerfile
@@ -20,7 +20,7 @@ ENV BITNAMI_IMAGE_VERSION=10.1.14-r2 \
     BITNAMI_APP_NAME=mariadb \
     BITNAMI_APP_USER=mysql
 
-RUN bitnami-pkg unpack mariadb-10.1.14-2 --checksum 2d70b08d8289b71695f3bb5a9517129dd1449e465896ff7e2441b9027e66009a
+RUN bitnami-pkg unpack mariadb-10.1.14-3 --checksum 261d55ed7759cc6708750ff3baa84365f9b00473b7673868d12ac03875ae9823
 ENV PATH=/opt/bitnami/$BITNAMI_APP_NAME/sbin:/opt/bitnami/$BITNAMI_APP_NAME/bin:$PATH
 
 COPY rootfs/ /

--- a/test.sh
+++ b/test.sh
@@ -279,6 +279,34 @@ cleanup_environment
   [[ "$output" =~ "name: Marko" ]]
 }
 
+@test "Can setup master/slave replication using a custom master port" {
+  container_create default -d -p 3306 \
+    -e MARIADB_REPLICATION_MODE=master \
+    -e MARIADB_REPLICATION_USER=$MARIADB_REPLICATION_USER \
+    -e MARIADB_REPLICATION_PASSWORD=$MARIADB_REPLICATION_PASSWORD \
+    -e MARIADB_ROOT_PASSWORD=$MARIADB_PASSWORD \
+    -e MARIADB_DATABASE=$MARIADB_DATABASE
+
+  MASTER_HOST=$(container_exec default ip route list | grep ^default | awk '{print $3}')
+  MASTER_PORT=$(docker port $CONTAINER_NAME-default 3306/tcp | cut -d':' -f2)
+  container_create slave0 -d \
+    -e MARIADB_REPLICATION_MODE=slave \
+    -e MARIADB_REPLICATION_USER=$MARIADB_REPLICATION_USER \
+    -e MARIADB_REPLICATION_PASSWORD=$MARIADB_REPLICATION_PASSWORD \
+    -e MARIADB_MASTER_HOST=$MASTER_HOST \
+    -e MARIADB_MASTER_PORT=$MASTER_PORT \
+    -e MARIADB_MASTER_PASSWORD=$MARIADB_PASSWORD \
+    -e MARIADB_ROOT_PASSWORD=$MARIADB_PASSWORD \
+    -e MARIADB_DATABASE=$MARIADB_DATABASE
+
+  mysql_client default -uroot -p$MARIADB_PASSWORD $MARIADB_DATABASE -e \
+    "CREATE TABLE users (id INT AUTO_INCREMENT, name VARCHAR(30), datum TIMESTAMP DEFAULT CURRENT_TIMESTAMP, PRIMARY KEY(id)); \
+     INSERT INTO users(name) VALUES ('Marko');"
+
+  run mysql_client slave0 -uroot -p$MARIADB_PASSWORD $MARIADB_DATABASE -e "SELECT * FROM users\G"
+  [[ "$output" =~ "name: Marko" ]]
+}
+
 @test "Can setup master/slave replication with password for replication user" {
   container_create default -d \
     -e MARIADB_REPLICATION_MODE=master \

